### ISSUE TITLE
dyld: Library not loaded

### ISSUE THREAD

**Issue by kenmhaggerty** [AuthorizeNet/inperson-sdk-ios#41]
Hello – [This issue](https://github.com/AuthorizeNet/sdk-ios/issues/33) was closed by @brianmc on May 6 but I am still encountering it and can't figure out how to re-open the issue myself.

Here is the exact output I receive when I attempt to run a sample app with the Authorize.Net CocoaPod added to the Podfile but with no application code whatsoever:

```
dyld: Library not loaded: @rpath/XCTest.framework/XCTest
  Referenced from: /Users/kenmhaggerty/Library/Developer/CoreSimulator/Devices/19B60B49-7EDB-4A89-8F8B-4DB32AB095AD/data/Containers/Bundle/Application/2F5C6EAE-0CE6-40C4-9636-A051CCE2DD4C/MyProject.app/MyProject
  Reason: image not found
```

Not that the above error message is only printed to the console when the app is run; simply building the app will not cause any error.

Here are the contents of my Podfile, trying to make it as simple as possible:

```
target 'MyProject'
pod 'authorizenet-sdk'
```

MyProject is a mixed Objective-C and Swift project. I am using Xcode 7.3.

I have tried adding libxml2.dylib to "Link Binary with Libraries" build phase as recommended in the CocoaPods documentation but the issue still persists.

Please let me know what other details I can include to help resolve this issue.

**Comment by brianmc** [comment]
Hi @kenmhaggerty,
We're trying to repro here.  I will also say that the next version will be released shortly and it will be a framework so we're hopeful that it should be much easier to integrate.

Brian

**Comment by Koteswar97** [comment]
Hi I am facing the same issue dyld: Library not loaded 
plz help me

**Comment by jorgevmendoza** [comment]
Hi there!, am having exactly the same issue and i haven't been able to solve it, i'd try many ways to specify the new path for the XCTest framework but it doesn't seem to get the right new address, any suggestions for a quick fix before releasing a new version?  

Any help would be much appreciated!

Regards.

Jorge.

Xcode 7.3.1
Target iOS 9.3
Framework installed over Cocoapods: 0.36.0

**Comment by ptaneja** [comment]
Please try removing reference of class Anet_SDKTests from your workspace. It should work.

**Comment by jorgevmendoza** [comment]
@ptaneja Thanks for your response, i toke your advise and did comment the import of **Anet_SDKTests** in the file authorizenet-sdk-umbrella.h but i keep getting the same error once i run the app:

dyld: Library not loaded: @rpath/XCTest.framework/XCTest
  Referenced from: /Users/jorgemendoza/Library/Developer/CoreSimulator/Devices/BE075DF5-24A1-4549-86BF-A221981AFA3A/data/Containers/Bundle/Application/05484C5F-239B-4B9F-AACC-14C22E17ACAA/Apoyo Seguro.app/Frameworks/authorizenet_sdk.framework/authorizenet_sdk
  Reason: image not found

Thanks for your time!

**Comment by ptaneja** [comment]
@jorgevmendoza @kenmhaggerty @Koteswar97

Could you please try the now the latest version 2.0.7. Please delete your Cocoapods cache under Library and remove authorize.net pods from your workspace and then try.

**Comment by ptaneja** [comment]
Closing this! Fixed in 2.0.7

**Comment by jorgevmendoza** [comment]
Totally worked! thanks guys!
